spec/classes/init_spec.rb

```diff
@@ -532,20 +447,20 @@
       var[:name].each do |var_name|
         var[:valid].each do |valid|
           context "with #{var_name} (#{type}) set to valid #{valid} (as #{valid.class})" do
-            let(:params) { validation_params.merge(:"#{var_name}" => valid) }
+            let(:params) { validation_params.merge("#{var_name}": valid) }
 
             it { is_expected.to compile }
           end
         end
 
         var[:invalid].each do |invalid|
           context "with #{var_name} (#{type}) set to invalid #{invalid} (as #{invalid.class})" do
-            let(:params) { validation_params.merge(:"#{var_name}" => invalid) }
+            let(:params) { validation_params.merge("#{var_name}": invalid) }
 
             it 'fails' do
-              expect {
+              expect do
                 catalogue
-              }.to raise_error(Puppet::Error, %r{#{var[:message]}})
+              end.to raise_error(Puppet::Error, %r{#{var[:message]}})
```

Comment:
Could also use a `compile.and_raise_error`

Reply:
i though i had all of them by now.... damn